migrator/src/main/java/uk/nhs/digital/ps/migrator/model/hippo/MappedFields.java

```diff
@@ -0,0 +1,186 @@
+package uk.nhs.digital.ps.migrator.model.hippo;
```

Comment:
I'm not sure how this file is related to your PR :(

Reply:
@wojtek-oledzki Good catch - this isn't supposed to be in, I'll be removing it altogether